multimodal/src/autogluon/multimodal/optimization/lit_distiller.py

```diff
@@ -218,6 +242,12 @@ def _compute_loss(
         )
         loss += soft_label_loss * self.soft_label_weight
 
+        embedding_loss = self._compute_embedding_loss(
```

Comment:
The current implementation won't work if the features from teacher / student have different dimensionalities.